modules/microshift-manifests-override-paths.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-applications.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-manifests-override-paths_{context}"]
+= Override the list of manifest paths
+You can override the list of default paths with the following procedure.
+
+.Procedure
+
+. You can set `manifests.kustomizePaths` to `++"++/opt/alternate/path++"++` in the configuration file for a single path. Alternativly, you can set `kustomizePaths` to `++"++/opt/alternative/path.d/++*"++` in the configuration file for a glob pattern.
++
+[source,terminal]
+----
+manifests:
+    kustomizePaths:
+        - <location> <1>
+----
+<1> Set the location to `++"++/opt/alternate/path++"++` for single path and `++"++/opt/alternative/path.d/++*"++` for a glob pattern.
```

Comment:
```suggestion
<1> Set each location entry to an exact path (`++"++/opt/alternate/path++"++`) or a glob pattern (`++"++/opt/alternative/path.d/++*"++`).
```

I'm sure that doesn't fit our style guide, but I hope you get the idea.

Reply:
Yeah, the style guide isn't a big fan of the use of parenthesis. I did update to "using".